tests/test_models.py

```diff
@@ -146,13 +146,13 @@ def test_rectangular_to_polar_dict(self, basic_stim):
         energy, state = po.simul.non_linearities.rectangular_to_polar_dict(y, residuals=True)
         y_hat = po.simul.non_linearities.polar_to_rectangular_dict(energy, state, residuals=True)
         for key in y.keys():
-            assert torch.norm(y[key] - y_hat[key]) < 1e-5
+            assert torch.linalg.norm(y[key] - y_hat[key]) < 1e-5
```

Comment:
```suggestion
            diff = y[key] - y_hat[key]
            assert torch.linalg.vector_norm(diff.flatten(), ord=2) < 1e-5
```
Let's make this a bit more readable, since this is behavior that torch is doing.